test/acceptance/registry.js

```diff
@@ -167,7 +167,7 @@ describe('registry', function(){
       it('should return the component with custom headers', function() {
         expect(result[0].response.version).to.equal('1.0.0');
         expect(result[0].response.name).to.equal('hello-world-custom-headers');
-        expect(result[0].response.headers).to.be.deep.equal({'cache-control': 'public max-age=3600', 'test-header': 'Test-Value'});
+        expect(result[0].headers).to.be.deep.equal({'cache-control': 'public max-age=3600', 'test-header': 'Test-Value'});
```

Comment:
I actually think I didn't pay so much attention on the previous review as I was surprised to see this. What do you think @ivan-dimitrov-skyscanner ? Can we safely change this and consider it as a bugfix?